api/app/dao/provider_details_dao.py

```diff
@@ -27,20 +27,19 @@ def get_provider_details_by_identifier(identifier):
 
 def get_alternative_sms_provider(identifier):
     alternate_provider = None
-    if os.getenv('FEATURE_SAP_ENABLED'):
-        if identifier == 'sap':
-            alternate_provider = 'telstra'
-        elif identifier == 'telstra':
-            alternate_provider = 'sap'
-        # We need to have a third check here because our tests expect that
-        # Twilio is enabled.
-        elif identifier == 'twilio':
-            alternate_provider = 'sap'
+    if identifier == 'telstra':
```

Comment:
can we change the logic in here a bit? something like:

```
if identifier != 'sap':
  alternate_provider = 'sap'
else:
  alternate_provider = 'telstra'
```

Reply:
I was not sure whether to remove Twilio case here. I will change the logic and update.